crud/common/map_call_cases/batch_insert_iter.lua

```diff
@@ -70,7 +70,7 @@ function BatchInsertIterator:get()
     local replicaset = self.next_index
     local func_args = {
         self.space_name,
-        self.next_batch,
+        self.next_batch.tuples,
```

Comment:
This change regards to `upsert_many` because upsert requires some changes for `sharding.split_tuples_by_replicaset` function, that's why `sharding.split_tuples_by_replicaset` output has different format after `upset_many` implementation.